Aptoide/aptoide-client-v8
# [AN-1784] Sharing app related Posts with the StoreId from the shared …

What does this PR do?

   This pull-request adds the storeId of the shared app store to the Share card body. This happens in  all App related cards share (Recommendation/Social Install/Social Recommendation/PopularApp) apart from AppUpdate (because is always the latest) 

Where should the reviewer start?

- [ ] Timeline [line 105] -> check if AppPost comes with StoreId.

How should this be manually tested?

  Open Aptoide v8 > Timeline > Share an App card and check in charles if the request body goes with StoreId (store id of the store where the app from the card is)

What are the relevant tickets?

Tickets related to this pull-request: [AN-1784](https://aptoide.atlassian.net/browse/AN-1784)

Screenshots (if appropriate):
N/A

Questions:

   Is there a blog post? N/A
   Does this add new dependencies which need to be added? No
